.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,44 @@
+name: "\U0001F41E Bug report"
+description: Report an issue with Sheepdog
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a link to a repo or REPL that can reproduce the problem you ran into. If a report is provided without an easy way to reproduce the bug, it's likely we will not be able to help.
+      placeholder: Reproduction
+    validations:
+      required: true
+  - type: dropdown
+    id: severity
+    attributes:
+      label: Severity
+      description: Select the severity of this issue
+      options:
+        - annoyance
```

Comment:
```suggestion
        - annoyance
        - serious but i have a workaround
        - blocking me on a specific feature
```

Reply:
@paoloricciuti, the "I" should be capitalised here

Reply:
FFS I will never get the habit of doing this